angr/tablespecs.py

```diff
@@ -1,5 +1,5 @@
 import claripy
-
+# TODO: When is a table not a table? ... When it's a chair (support variable byte widths)
```

Comment:
this is kind of complicated because this class is designed to function when there's no state yet. The easiest thing to do would probably be to add byte width to the constructor.